### PR TITLE
switching eks prod to gevent

### PR DESCRIPTION
**Pull request by duallain**

### CODE REVIEW

File: mozmeao-fr/bedrock-prod/deploy.yaml [hunk 1]
```diff
@@ -113,6 +113,8 @@ spec:
               name: bedrock-prod-secrets
         - name: GTM_CONTAINER_ID
           value: GTM-MW3R8V
+        - name: GUNICORN_WORKER_CLASS
+          value: gunicorn.workers.ggevent.GeventWorker
```
Comment (pmac) [inline mozmeao-fr/bedrock-prod/deploy.yaml:117]: According to [the docs](https://docs.gunicorn.org/en/stable/install.html#async-workers) we should be able to just say `gevent` here, but before that we have to install `gevent` into the bedrock image, which we aren't currently doing.

Comment (pmac) [reply mozmeao-fr/bedrock-prod/deploy.yaml:117]: NM. Found your `gevent` branch.

Comment (pmac) [reply mozmeao-fr/bedrock-prod/deploy.yaml:117]: But you can just say `gevent` here. It's the same thing.

https://github.com/benoitc/gunicorn/blob/61ccfd6c38d477a908e0f376757bbb884438053a/gunicorn/workers/__init__.py#L10

Comment (duallain) [reply mozmeao-fr/bedrock-prod/deploy.yaml]: @pmac done across all branches, did test it with the test deploy first and it worked.

**Pull request closed without merging**